config.go

```diff
@@ -93,7 +95,11 @@ func ConfigStringToClusterConfig(configString string) (*gocql.ClusterConfig, err
 				if len(settingSplit) != 2 {
 					return nil, fmt.Errorf("missing =")
 				}
-				key, value := strings.TrimSpace(settingSplit[0]), strings.TrimSpace(settingSplit[1])
+				key, rawValue := strings.TrimSpace(settingSplit[0]), settingSplit[1]
+				value, err := url.QueryUnescape(rawValue)
```

Comment:
What do you think about moving QueryUnescape to only the values that need it?
username, password, certPath, keyPath, and caPath

Reply:
Please make sure to add QueryUnescape fail tests for each one